packages/eva/mapping.json

```diff
@@ -1124,6 +1124,356 @@
+          },
+          "paddingHorizontal": {
+            "type": "number"
+          },
+          "backgroundColor": {
+            "type": "string"
+          },
+          "multiSelectBackgroundColor": {
+            "type": "string"
+          },
+          "multiSelectTextColor": {
+            "type": "string"
+          },
+          "textMarginHorizontal": {
+            "type": "number"
+          },
+          "textFontSize": {
+            "type": "number"
+          },
+          "textLineHeight": {
+            "type": "number"
+          },
+          "textFontWeight": {
+            "type": "string"
+          },
+          "textColor": {
+            "type": "string"
+          }
+        },
+        "appearances": {
+          "default": {
+            "default": true
+          }
+        },
+        "variantGroups": {},
+        "states": {
+          "disabled": {
+            "default": false,
+            "priority": 0,
+            "scope": "all"
+          },
+          "active": {
+            "default": false,
+            "priority": 1,
+            "scope": "all"
+          },
+          "selected": {
+            "default": false,
+            "priority": 2,
+            "scope": "all"
+          }
+        }
+      },
+      "appearances": {
+        "default": {
+          "mapping": {
+            "paddingHorizontal": 8,
+            "paddingVertical": 8,
+            "textMarginHorizontal": 8,
+            "backgroundColor": "background-basic-color-1",
+            "multiSelectBackgroundColor": "background-basic-color-1",
+            "multiSelectTextColor": "background-alternative-color-1",
+            "textFontSize": "text-subtitle-1-font-size",
+            "textFontWeight": "text-subtitle-1-font-weight",
+            "textLineHeight": "text-subtitle-2-line-height",
+            "textColor": "background-alternative-color-1",
+            "state": {
+              "active": {
+                "backgroundColor": "background-basic-color-2"
+              },
+              "selected": {
+                "textColor": "text-alternate-color",
+                "backgroundColor": "color-primary-default"
+              },
+              "disabled": {
+                "textColor": "text-hint-color",
+                "multiSelectTextColor": "text-hint-color"
+              }
+            }
+          }
+        }
+      }
+    },
+    "DropdownGroup": {
+      "meta": {
+        "scope": "all",
+        "parameters": {
+          "backgroundColor": {
+            "type": "string"
+          },
+          "itemPaddingLeft": {
+            "type": "number"
+          }
+        },
+        "appearances": {
+          "default": {
+            "default": true
+          }
+        },
+        "variantGroups": {},
+        "states": {}
+      },
+      "appearances": {
+        "default": {
+          "mapping": {
+            "backgroundColor": "background-basic-color-1",
+            "itemPaddingLeft": 20
```

Comment:
I guess this should be horizontal padding for all sub-components

Reply:
Refactored